olive/cache.py

```diff
@@ -105,12 +105,10 @@ def get_run_json_path(
         """Get the path to the run json."""
         pass_config_hash = hash_dict(pass_config)[:8]
         if not accelerator_spec:
-            run_json_path = self.dirs.runs / f"{pass_name}-{input_model_number}-{pass_config_hash}.json"
+            run_json_path = hash_string(f"{pass_name}-{input_model_number}-{pass_config_hash}")[:8]
```

Comment:
Will this bring inconvininence to debug/look up specific pass results in the cache folder?

Reply:
Rather than `[:8]` pass a parameter to the `hash_string` to generate a fixed length hash string.

Reply:
I agree with Jiapeng's comment. Also, we currently have this feature where we can clean up the results of specific pass (that we might be testing) and its children output models https://github.com/microsoft/Olive/blob/7ab3f92a7b3151a25042067106644ebf2b4c9448/olive/cache.py#L115 

This feature depends on the name being part of the cache name. 

Is there really a need to shorten the cache path length. I think using a shorter hash than `8` might be sufficient if it's really needed.

Reply:
The basic idea is no user would care the intermediate models, as long as the output model path is correct. @jambayk is that `clean_pass_run_cache` used in our core logic?

Reply:
yes https://github.com/microsoft/Olive/blob/0a57b9d0095059a8ca8775e119ca540579405c82/olive/engine/engine.py#L120